java/vector/src/main/java/org/apache/arrow/vector/BaseIntVector.java

```diff
@@ -26,4 +26,9 @@
    * set value at specific index, note this value may need to be need truncated.
    */
   void setWithPossibleTruncate(int index, long value);
+
+  /**
+   * get value at specific index, note this value may haven been extended to long.
```

Comment:
```suggestion
   * Get value at specific index, note this value may haven been extended to long.
```

Reply:
You might want to comment about throwing an NPE if the value is null and provider a pointer to turn off the additional checks?

Reply:
Yes, will throw NPE, what do you mean by "provider a pointer to turn off the additional checks"

Reply:
A reference to NullCheckingForGet

Reply:
Thanks, fixed.